M1_Sample_Code/2_hello_world_parameterized/configs/2_hello_world_config_config_1.yaml

```diff
@@ -0,0 +1 @@
+sample_param: do this thing
```

Comment:
lets drop these files from the PR - they will get generated when running the code